File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < 2 * n + 2; j++)
        {
            if (j == n || j == n + 1)
            {
                printf(" ");
            }

            else if (j + 1 >= n - i & j < n)
            {
                printf("#");
            }

            else if (j - 2 <= n + i & j > n)
            {
                printf("#");
            }

            else if (j < n)
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break

for i in range(h):
    blank = h - i - 1
    hash = i + 1
    print(" " * blank + "#" * hash + "  " + "#" * hash)
